test/reachsets.jl

```diff
@@ -102,4 +102,27 @@ end
     @test isequivalent(set(overapproximate(c, Zonotope)), set(R))
     @test isequivalent(set(overapproximate(d, Zonotope)), set(R))
 
+    order = 4
```

Comment:
perhaps wrap this into its own test set, `@testset "Taylor model reach-sets with non-float coefficients" begin

Reply:
see [CI log](https://github.com/JuliaReach/ReachabilityAnalysis.jl/runs/6041561496?check_suite_focus=true#step:6:618) for other errors; in particular, we need to make all the necessary `using`'s

Reply:
Thanks!